contrib/templates/data-engineering/assets/etl-pipeline/databricks_template_schema.json

```diff
@@ -0,0 +1,46 @@
+{
+    "welcome_message": "\nWelcome to the data-engineering pipeline template!",
+    "properties": {
+        "pipeline_name": {
+            "type": "string",
+            "description": "\nPlease provide the name of the pipeline to generate.\npipeline_name",
+            "default": "etl_pipeline",
+            "order": 1
+        },
+        "format": {
+            "type": "string",
+            "description": "\nPlease select the format to use to define this pipeline.\nformat",
+            "order": 2,
+            "enum": [
+                "python files",
+                "sql files",
+                "notebooks"
+            ],
+            "default": "python files"
+        },
+        "only_python_files_supported": {
+            "skip_prompt_if": {
+                "properties": {
+                    "format": {
+                        "pattern": "python files"
+                    }
+                }
+            },
+            "default": "ignored",
+            "type": "string",
+            "description": "{{fail Only Python files are supported in this template at this time.}}",
```

Comment:
Not a string; template fails.

Reply:
Will fix. It does fail, as it should, but the error indeed wasn't very nice :)